source/Octopus.Tentacle.Client/Execution/RpcCallExecutor.cs

```diff
@@ -103,86 +109,68 @@ internal class RpcCallExecutor
             RpcCall rpcCall,
             Func<CancellationToken, Task<T>> action,
             ILog logger,
-            bool abandonActionOnCancellation,
             ClientOperationMetricsBuilder clientOperationMetricsBuilder,
             CancellationToken cancellationToken)
         {
-            return await rpcCallNoRetriesHandler.ExecuteWithNoRetries(
-                    async ct =>
-                    {
-                        // Wrap the action in a task so it doesn't block on sync Halibut calls, and cancellation token is respected.
-                        return await Task.Run(async () => 
-                        {
-                            var rpcCallMetricsBuilder = RpcCallMetricsBuilder.StartWithoutRetries(rpcCall);
-                            var start = DateTimeOffset.UtcNow;
+            // Wrap the action in a task so it doesn't block on sync Halibut calls, and cancellation token is respected.
```

Comment:
Just wondering, is this comment still legit now? I.e., can we remove it, as well as wrapping it in a task?

It might be more retesting than we have appetite for, but thought I'd ask.

Reply:
The comment is out of date, I'll try and remove the Task.Run wrappers and see what happens. Hopefully they are now redundant!!

Reply:
Green build with these removed, they are redundant 🥳